commands/Jetpack_Plugin_Search.php

```
@@ -234,5 +343,84 @@ private function is_version_match( \stdClass $plugin_data ): bool {
+			$headers
+		);
+
+		if ( ! empty( $this->export_excluded_columns ) ) {
+			$this->export_excluded_columns = array_map(
+				static fn ( $column ) => strtoupper( preg_replace( '/\s+/', '', $column ) ),
+				$this->export_excluded_columns
+			);
+
+			foreach ( $this->export_excluded_columns as $column ) {
+				$column_index = array_search( $column, $json_header_compare, true );
+				$column_name  = $headers[ $column_index ];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$column_name'. (Unused Code Rules)

(UnusedLocalVariable)